src/target_intacct/const.py

```diff
@@ -28,9 +30,10 @@
     "customers": "CUSTOMER",
     "classes": "CLASS",
     "locations": "LOCATION",
-    "departments": "DEPARTMENT"
+    "departments": "DEPARTMENT",
+    "employees": "EMPLOYEE",
 }
 
 GET_BY_DATE_FIELD = "WHENMODIFIED"
```

Comment:
Is this variable used anywhere now that we've removed the import? Any time you remove the last reference to a thing you should also probably remove the thing.